app/models/dwp_override.rb

```diff
@@ -1,2 +1,5 @@
 class DWPOverride < ApplicationRecord
+  belongs_to :legal_aid_application
+
+  validates :passporting_benefit, :evidence_available, presence: true
```

Comment:
I think this can be removed. These are added over two separate pages according to the design, so we are planning to validate using the form model on each of those pages there instead.

Reply:
Agreed

Reply:
Agreed